templates/demo-store/app/lib/utils.ts

```diff
@@ -2,13 +2,13 @@ import {useLocation, useMatches} from '@remix-run/react';
 import type {MoneyV2} from '@shopify/hydrogen/storefront-api-types';
 import typographicBase from 'typographic-base';
 
+import {useRootLoaderData} from '~/root';
```

Comment:
Hmmm with this change my eslint is complaining a bunch
![image](https://github.com/Shopify/hydrogen/assets/1019769/2ef6fca2-3f81-436d-bf63-aa4bc1d9f350)

The weird thing is, this didn't break the ci checks. That leads me to think that something isn't set up properly to run eslint in ci?